### DISCUSSION TITLE
Tags Autofill/Search

### DISCUSSION THREAD

**Discussion by boundingstar** [koalagator/koalagator#21] [Ideas / Feature Requests]
If the tags field in event creation had a search/autofill feature (as what is used for the venue field), this would prevent as many duplicate tags from being created and splitting tag usage between multiple slight variants. Currently filtering events by tag misses relevant events due to them having different tag spellings

**Comment by evolve2k** [comment]
Oh yeah this is a great idea!
(Could be a nice opportunity for someone to add a dynamic lookup, likley using hotwire).

Currently I use the site in Firefox and double clicking the text input field brings up a firefox list of past entries that I've made.
Not a fix but maybe can help you save effort in a small way until such time as this change is implemented.
